src/Components/WeatherDetails/WeatherDetails.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import './WeatherDetails.scss';
+
+const WeatherDetails = ({ apiRequestData }) => {
+  const { cod } = apiRequestData;
+  const locationNotFound = cod !== 200 ? true : false;
```

Comment:
this is wrong, you can just check if you have the data and make an "early return"
```

if(!data) {
    return null
}


return ( 
<div> 
....data goes here
</div>
```

Reply:
Can't use that solution cuz API have results if is empty it returns to element in obj, I can only match that by response code. Thats why first I have destruct code and made ternary operator